### PR TITLE
urls

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/post/2019-04-25-r-devel-linux-x86-64-debian-clang.markdown [hunk 1]
```diff
@@ -11,7 +11,7 @@ In this post, we shall introduce CRAN checks in general and use the recent chang
 
 # CRAN checks 101
 
-All CRAN packages are _R CMD Check_-ed regularly on 12 CRAN platforms called [_CRAN Package Check Flavors_](https://cran.r-project.org/web/checks/check_flavors.htmlac), more platforms than at submission. The results of these checks are reported on each package's check results page, cf [e.g. `fpeek`'s one](https://cran.r-project.org/web/checks/check_results_fpeek.html). 
+All CRAN packages are _R CMD Check_-ed regularly on 12 CRAN platforms called [_CRAN Package Check Flavors_](https://cran.r-project.org/web/checks/check_flavors.html), more platforms than at submission. The results of these checks are reported on each package's check results page, cf [e.g. `fpeek`'s one](https://cran.r-project.org/web/checks/check_results_fpeek.html).
```
Comment (gaborcsardi) [inline content/post/2019-04-25-r-devel-linux-x86-64-debian-clang.markdown]: I guess this does not really belong here per se. :)

File: content/post/2019-12-10-pkg-urls.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+Why put URLs in DESCRIPTION?
+
+* It will help your users find your package's pretty documentation from the CRAN page, instead of just the less pretty PDF manual.
+
+* Likewise, from the CRAN page your contributors can directly find where to submit patches. 
+
+* If your package has a package-level man page, and it should (e.g. as drafted by [`usethis::use_package_doc()`](https://usethis.r-lib.org/reference/use_package_doc.html) and then generated by [`roxygen2`](https://roxygen2.r-lib.org/articles/rd.html#packages)), then after typing say `library("rhub")` and then `?rhub`, your users will find the useful links.
+
+* Other tool such as [`helpdesk`](https://github.com/yonicd/helpdesk) and [`pkgsearch` RStudio addin](https://r-hub.github.io/pkgsearch/reference/index.html#section-rstudio-addin) can help surface the URLs you store in DESCRIPTION.
```
Comment (gaborcsardi) [inline content/post/2019-12-10-pkg-urls.Rmarkdown]:
```suggestion
* Other tools such as [`helpdesk`](https://github.com/yonicd/helpdesk) and the [`pkgsearch` RStudio addin](https://r-hub.github.io/pkgsearch/reference/index.html#section-rstudio-addin) can help surface the URLs you store in DESCRIPTION.
```

File: content/post/2019-12-10-pkg-urls.Rmarkdown [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+> Quick tip, you can add GitHub URLs (URL and BugReports) to DESCRIPTION by running [`usethis::use_github_links()`](https://usethis.r-lib.org/reference/use_github_links.html). :rocket:
+
+## Where else put your URLs?
+
+For the same reasons as previously, you should make the most of all places that can store your package's URL(s). Have you put your package's docs URL
+
+* in [the pkgdown config file](https://pkgdown.r-lib.org/reference/build_site.html#yaml-config), if that's how you built it?
+
+* in the GitHub repo URL field, or the equivalent for your development platform?
```
Comment (gaborcsardi) [inline content/post/2019-12-10-pkg-urls.Rmarkdown]: I didn't immediately know what this was, but maybe that's just me.

File: content/post/2019-12-10-pkg-urls.Rmarkdown [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+Have you used any of your package's URLs
+
+* [In your public message about your package, e.g. as an answer to someone's question](https://community.rstudio.com/t/pkgdown-site-seo/26706/6)?
+
+* In the slides of [your talk about the package](https://www.tidyverse.org/blog/2018/07/carpe-talk/)?
+
+Don't miss any opportunity to point users and contributors in the right direction!
+
+## What URLs in DESCRIPTION files of CRAN packages?
```
Comment (gaborcsardi) [inline content/post/2019-12-10-pkg-urls.Rmarkdown]: What URLs _do people use_... ? or something like that?

File: content/post/2019-12-10-pkg-urls.Rmarkdown [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+```{r domain}
+dplyr::count(parsed_db, domain, sort = TRUE)
+
+```
+
+GitHub seems to be the most popular development platform, as least from this sample of CRAN packages that indicate an URL. It is also possible that some developers set up their own GitLab server with a own domain. 
+Many packages link to `www.r-project.org` which is not very informative, or to their own CRAN page which can be informative. 
+
+> By the way if you visit CRAN pages to get package metadata such as dependencies or reverse dependencies, you could also use [our `pkgsearch` package](/2019/11/29/pkgsearch/).
```
Comment (gaborcsardi) [inline content/post/2019-12-10-pkg-urls.Rmarkdown]: Hmmm, this dependency part seems to be a bit out of place to me. Is it actually needed for the rest of post?

File: content/post/2019-04-25-r-devel-linux-x86-64-debian-clang.markdown (hunk 1 above)
Comment (maelle) [reply content/post/2019-04-25-r-devel-linux-x86-64-debian-clang.markdown]: oops, indeed, thanks!

File: content/post/2019-12-10-pkg-urls.Rmarkdown (hunk 5 above)
Comment (maelle) [reply content/post/2019-12-10-pkg-urls.Rmarkdown]: right, no. btw I opened an issue in pkgsearch about that.

File: content/post/2019-12-10-pkg-urls.Rmarkdown (hunk 3 above)
Comment (maelle) [reply content/post/2019-12-10-pkg-urls.Rmarkdown]: good point, especially as it's not obvious where that is in the docs!

Comment (gaborcsardi) [reply content/post/2019-12-10-pkg-urls.Rmarkdown]: This is the URL at the top of the repo page, right? Maybe we can have a screenshot here, or just write "at the top right of the repo page" or sth similar?

**Pull request merged**